config.toml.in

```diff
@@ -35,6 +35,17 @@ type = "file"
   # The number of backups to keep.
   # Specify -1 to allow unlimited backups, 0 to not create backups at all.
   #max = 100
+  
+# type = "s3"
```

Comment:
You should also duplicate the whole `storage.backup` settings from above so they can be different per backend, i.e.:

```
  #[storage.backup]
  # Whether to auto-create backups before the safe is modified.
  #auto = true
  # The number of backups to keep.
  # Specify -1 to allow unlimited backups, 0 to not create backups at all.
  #max = 100
```